### PR TITLE
Xpk fix parallel node pool create

### PR DESCRIPTION
**Pull request by Obliviour**
## Fixes / Features
- the ability to do parallel node pool creation. `with` breaks this by waiting for the popen instance to close before moving on.

## Testing / Documentation

- [ y] Parallel Node Pool Creation works.

### CODE REVIEW

**Comment by Obliviour** [comment]
Note there are commits from another PR that will get merged first and will clean this PR up afterward.

**Review by rwitten — APPROVED** [review]
I'm conflicted as discussed in chat.

**Review by rwitten — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -62,22 +62,25 @@ might want to use `Cluster List` or `Workload List` commands. Finally, you can
 cleanup with a `Cluster Delete`.
 
 ## Cluster Create
-*   Cluster Create (provision on-demand capacity):
```
Comment (rwitten) [inline README.md:65]: Wait is this the CR you meant to put here? Doesn't seem to have anything to do with the title:
```
Xpk fix parallel node pool create #27
```

**Review by rwitten — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Obliviour) [reply README.md]: I build this change on top of the capacity arguments one purposefully for testing the two features together lol. Merged the capacity changes in and this PR looks clean now.

**Review by rwitten — APPROVED** [review]

**Pull request merged**